### DISCUSSION TITLE
Is prisma have type casting

### DISCUSSION THREAD

**Discussion by Bennison-Devadoss** [prisma/prisma#15779] [Q&A]

**Comment by nurul3101** [comment]
Hi @Bennison-Devadoss 👋 
Can you please elaborate on the use case?
If you are using Raw Query then Prisma casts these database types to appropriate javascript types: [Reference](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-query-type-mapping)
<img width="692" alt="Screenshot 2022-10-13 at 6 31 37 PM" src="https://user-images.githubusercontent.com/13049676/195603396-6effa403-79d4-432e-9d33-66294fc0b100.png">

**Reply by vladi-stevanovic** [reply]
👋 Hello @Bennison-Devadoss did you perhaps have a chance to check Nurul's recommendation? 
Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
